packages/account-v2/src/utils/personalDetailsUtils.ts

```diff
@@ -0,0 +1,188 @@
+    }
+
+    if (accountSettings.request_professional_status) {
+        initialValues.requestProfessionalStatus = accountSettings.request_professional_status;
+    }
+
+    return initialValues;
+};
+
+export const getPersonalDetailsBaseValidationSchema = () => {
+    const characterLengthMessage = 'You should enter 2-50 characters.';
+    const addressLengthMessage = 'Should be less than 70 characters.';
+    const phoneNumberLengthMessage = 'You should enter 9-35 numbers.';
+
+    const regexPattern = ValidationConstants.patterns;
+
+    return Yup.object({
+        accountOpeningReason: Yup.string().required('Account opening reason is required.'),
+        addressCity: Yup.string()
+            .required('Town/City is required.')
+            .max(70, addressLengthMessage)
+            .matches(regexPattern.addressCity, 'Only letters, space, hyphen, period, and apostrophe are allowed.'),
+        addressLine1: Yup.string()
+            .trim()
+            .required('First line of address is required.')
+            .max(70, addressLengthMessage)
+            .matches(regexPattern.address, 'Use only the following special characters: 70'),
+        addressLine2: Yup.string()
+            .trim()
+            .max(70, addressLengthMessage)
+            .matches(regexPattern.address, 'Use only the following special characters: 70'),
+        addressPostcode: Yup.string()
+            .max(20, 'Please enter a Postal/ZIP code under 20 chatacters.')
+            .matches(regexPattern.postalCode, 'Only letters, numbers, space, and hyphen are allowed.'),
+        citizenship: Yup.string().required('Citizenship is required.'),
+        countryOfResidence: Yup.string().required('Country of residence is required.'),
+        dateOfBirth: Yup.date().typeError('Please enter a valid date.').required('Date of birth is required.'),
+        employmentStatus: Yup.string().required('Employment status is required.'),
+        firstName: Yup.string()
+            .required('First name is required.')
+            .min(2, characterLengthMessage)
+            .max(50, characterLengthMessage)
+            .matches(regexPattern.name, 'Letters, spaces, periods, hyphens, apostrophes only.'),
+        lastName: Yup.string()
+            .required('Last Name is required.')
+            .min(2, characterLengthMessage)
+            .max(50, characterLengthMessage)
+            .matches(regexPattern.name, 'Letters, spaces, periods, hyphens, apostrophes only.'),
+        nameDOBConfirmation: Yup.boolean().required(),
+        phoneNumber: Yup.string()
+            .required('Phone number is required.')
+            .matches(regexPattern.phoneNumber, 'Please enter a valid phone number.')
+            .min(9, phoneNumberLengthMessage)
+            .max(35, phoneNumberLengthMessage),
+        placeOfBirth: Yup.string().required('Place of birth is required.'),
+        taxIdentificationNumber: Yup.string(),
```

Comment:
The validation for tax indentification number seems different from prod. <img width="925" alt="Screenshot 2024-03-25 at 10 45 24 PM" src="https://github.com/binary-com/deriv-app/assets/84661147/ea200dd3-60fb-482d-9a5e-a32864258b5e">

Reply:
It was previously inside the code base for personal-details-utils not sure why it's different from production 🙏  @likhith-deriv can you help identify this 🙏 

<img width="1344" alt="Screenshot 2024-03-26 at 10 11 17 AM" src="https://github.com/binary-com/deriv-app/assets/103026762/e3350a22-11d2-42c9-b92a-cb0ed511587b">

Reply:
Hi @suisin-deriv , it is a mistake. Please update it with the validation present in v1 - `packages/account/src/Sections/Profile/PersonalDetails/validation.ts`